ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/title/clientlibs/less/title.less

```diff
@@ -0,0 +1,48 @@
+/* title component styles */
+
+.cmp-title{
+font-family: sans-serif, arial;
+color:#333;
+font-weight:700;
+margin:20px 0 25px 0;
+line-height:1.7;
+padding:0;
+
+h1{
+&.cmp-title__text{
+font-size:36px;
+}
+}
+
+h2{
+&.cmp-title__text{
+font-size:32px;
+}
+}
+
+h3{
+&.cmp-title__text{
+font-size:28px;
+}
+}
+
+h4{
+&.cmp-title__text{
+font-size:24px;
+}
+}
+
+h5{
+&.cmp-title__text{
+font-size:20px;
+}
+}
+
+h6{
+&.cmp-title__text{
+font-size:16px;
+}
+}
+
+
+}
```

Comment:
@rakeshkumar-verma : p tag will also have to be added here

Reply:
As of now in this component we have one H1-H6 element in html markup.

Reply:
@rakeshkumar-verma : yes please add the p tag too in dropdown for title . it is required at various places

Reply:
added p tag

Reply:
default style will apply on p tag